Chart: update Overview (#7056)
DevExpress/devextreme-documentation: concepts/05 UI Components/Chart/02 Overview/00 Overview.md

DevExtreme Chart is an interactive UI component that visualizes data from local or remote storage with various [series types](/concepts/05%20UI%20Components/Chart/11%20Series%20Types/10%20Line%20Series.md '/Documentation/Guide/UI_Components/Chart/Series_Types/').

> In this overview, you can find a showcase of Chart elements, information about key features, and what to explore next.

Reviewer: ```suggestion
This overview highlights Chart elements, key features, and next steps.
```
